UI/app/src/main/java/com/example/pathfinder/Activities/PassengerDashboard.java

```diff
@@ -98,15 +107,13 @@ protected void onCreate(Bundle savedInstanceState) {
         mStopBtn.setChecked(update(KEY_STOP));
         mHandicapBtn.setChecked(update(KEY_HANDICAP));
 
-
-
         mStopBtn.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
```

Comment:
Nice job with the layout ! Looks good =)
I have a question regarding this method,
When clicking on the stop button and after on the accessibility request button, the labelling of the stop button is still "Cancel stop" but the stop is not asked anymore. And there is another inconsistency when we click on the stop and accessibility and cancel the stop, the accessibility is still on. 

Would it be maybe better to completely separate the stop and accessibility buttons so they are totally separate?

Reply:
Works like a charm !
🍾